x-pack/plugins/observability_solution/infra/public/alerting/common/components/threshold.test.tsx

```diff
@@ -64,7 +64,7 @@ describe('Threshold', () => {
 
     expect((Metric as jest.Mock).mock.calls[0][0].data[0][0]).toMatchInlineSnapshot(`
       Object {
-        "color": "#f8e9e9",
+        "color": "#BD271E",
```

Comment:
Won't this potentially be an issue for flakiness should there be any further changes to the design system or theming?

Reply:
I thought so, I will check if we can actually not pass the color for the test, or another option